main/components/UploadDocumentsForm.tsx

```diff
@@ -1,6 +1,8 @@
 import { useState, type FormEvent } from "react";
 import { useRouter } from 'next/navigation';
 import axios from "axios";
+import * as XLSX from 'xlsx';
```

Comment:
try `import { read } from "xlsx"`? I'm afraid of wildcard imports =)

Reply:
Fixed wildcard import :)